crates/biome_configuration/src/formatter.rs

```diff
@@ -98,6 +103,7 @@ impl Default for FormatterConfiguration {
             bracket_spacing: Default::default(),
             ignore: Default::default(),
             include: Default::default(),
+            use_editorconfig: true,
```

Comment:
I am a bit worried about this default. It makes sense to have to `true`, however this could break some formatting for users that already use Biome and have an `.editorconfig` somewhere. What do you think? Should we set it `false` for now, and set it to `true` for `v2`?

Reply:
That would be the most conservative. On the other hand, prettier enables editorconfig support by default, and keeping this behavior when migrating from prettier would be a better user experience.

I think we should set this to false by default, but when migrating from prettier, set it to true if it's unspecified.

ref: https://prettier.io/docs/en/configuration.html#editorconfig

Reply:
actually it turns out that prettier's `options.editorconfig` doesn't work at all, see: https://github.com/prettier/prettier/issues/15255

Reply:
Yeah, sounds good me!